### DISCUSSION TITLE
How do I merge two pull requests?

### DISCUSSION THREAD

**Discussion by adeemaamir240** [community/community#63663] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

When I hear here a Pull Shark badge that come to merge pull request so how to merge pull request

**Comment by nirav-gajera** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request

**Comment by adeemaamir240** [comment]
I will do these all step but my GitHub showing a error  : this merge request is very heavy

**Reply by lala-andy** [reply]
delete you repository and make another repository second follow this link https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request

**Comment by IceTheCoder** [comment, marked as answer]
Do you mean you have 2 separate pull requests and you want to merge both? Head to the Pull Requests tab on your repo, click on one of the pull requests and hit merge. Do this for the 2nd pull request as well.
